transport/http/handler.go

```diff
@@ -82,6 +88,26 @@ func (h handler) callHandler(w http.ResponseWriter, req *http.Request) error {
 
 	ctx := context.Background()
 
+	// Extract opentracing etc baggage from headers
+	var spanCtx opentracing.SpanContext
+	if req.Header != nil {
+		carrier := opentracing.HTTPHeadersCarrier(req.Header)
```

Comment:
you might be using a bit outdated version of opentracing, for http you want to use `HTTPHeaders` format, not `TextMap`.